doc/source/examples/dem/rectangular-hopper/rectangular-hopper.rst

```diff
@@ -0,0 +1,210 @@
+
+    subsection lagrangian physical properties
+        set gx                       = 0.0
+        set gy                       = -9.81
+        set gz                       = 0.0
+        set number of particle types = 1
+        subsection particle type 0
+            set size distribution type            = uniform
+            set diameter                          = 0.00224
+            set number                            = 40740
+            set density particles                 = 2500
+            set young modulus particles           = 1000000
+            set poisson ratio particles           = 0.3
+            set restitution coefficient particles = 0.94
+            set friction coefficient particles    = 0.2
+            set rolling friction particles        = 0.09
+        end
+        set young modulus wall           = 1000000
+        set poisson ratio wall           = 0.3
+        set friction coefficient wall    = 0.2
+        set restitution coefficient wall = 0.9
+        set rolling friction wall        = 0.09
+    end
+
+
+Model parameters
+~~~~~~~~~~~~~~~~~
+
+.. code-block:: text
+
+    subsection model parameters
+        set contact detection method                = dynamic
+        set dynamic contact search size coefficient = 0.9
+        set load balance method                     = frequent
+        set load balance frequency                  = 200000
+        set neighborhood threshold                  = 1.3
+        set particle particle contact force method  = hertz_mindlin_limit_overlap
+        set particle wall contact force method      = nonlinear
+        set integration method                      = velocity_verlet
+    end
+
+
+Simulation control
+~~~~~~~~~~~~~~~~~~
+
+.. code-block:: text
+
+    subsection simulation control
+        set time step        = 1e-5
+        set time end         = 7.5
+        set log frequency    = 1000
+        set output frequency = 1000
+        set output path      = ./output/
+        set output name      = hopper
+    end
+
+
+Floating walls
+~~~~~~~~~~~~~~
+
+Floating wall in this example is handled as explained in the `Silo example <../silo/silo.html>`_.
+
+.. code-block:: text
+
+    subsection floating walls
+        set number of floating walls = 1
+        subsection wall 0
+            subsection point on wall
+                set x = 0
+                set y = 0
+                set z = 0
+            end
+            subsection normal vector
+                set nx = 0
+                set ny = 1
+                set nz = 0
+            end
+        set start time = 0
+        set end time   = 4
+        end
+    end
+
+
+Running the simulation
+----------------------
+This simulation can be launched by
+
+.. code-block:: text
+
+  mpirun -np 8 dem_3d hopper.prm
+
+Post-precessing
+---------------
+A Python post-processing code is provided with this exemple. It allows to process the flow rate of particle with a module reading the output files.
```

Comment:
```suggestion
A Python post-processing code is provided with this example. It is used to measure the flow rate of particles with a module reading the output files.
```

Reply:
In this example, it is not clear that pyvista is used here in the postprocessing. A phrase solves this problem :)